schemas/plugins/common/unit.cue

```diff
@@ -0,0 +1,19 @@
+package common
+
+#unit: #timeUnit | #percentUnit | #decimalUnit
+
+#timeUnit: {
+	kind: "Milliseconds" | "Seconds" | "Minutes" | "Hours" | "Days" | "Weeks" | "Months" | "Years"
+}
+
+#percentUnit: {
+	kind:           "Percent" | "PercentDecimal"
+	decimal_places: number
+}
+
+#decimalUnit: {
+	#kind:          "Decimal"
+	decimal_places: number
+	suffix?:        "string"
+	unitDisplay?:   "short" | "long" | "narrow"
```

Comment:
perhaps we should name it `unit_display` to align the syntax on other fields.

Reply:
Or I could have just named it `display` since we're already inside of unit so it can be inferred (for context I took the values from [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#unitdisplay))

Reply:
ah that's a good idea. I will update it with this new name